docs/devops.md

```diff
@@ -7,82 +7,47 @@
 
-We welcome you to test these releases in a **"public beta testing"** mode and get early access to upcoming features to the platforms. Sometimes these features/changes are referred to as **next, beta, staging,** etc. interchangeably.
-
-Your contributions via feedback and issue reports will help us in making the production platforms at `freeCodeCamp.org` more **resilient**, **consistent** and **stable** for everyone.
-
-We thank you for reporting bugs that you encounter and help in making freeCodeCamp.org better. You rock!
-
-## Identifying the upcoming version of platform
+Once the developer team [`@freeCodeCamp/dev-team`](https://github.com/orgs/freeCodeCamp/teams/dev-team/members) is happy with the changes on the staging application, these changes are moved every few days to the [`production-current`](https://github.com/freeCodeCamp/freeCodeCamp/tree/production-current) branch.
 
-The domain name will be different than **`freeCodeCamp.org`**. Currently this public beta testing version is available at:
+This is the final release that pushes changes to our "production" platforms on freeCodeCamp.org
 
-<h3 align="center"><a href='https://www.freecodecamp.dev' _target='blank'><code>www.freecodecamp.dev</code></a></h4>
+### Testing changes - Integration and User Acceptance Testing.
 
-The dev-team merges changes from the `master` branch to `production-staging` when they release changes. Usually the top commit should be what you see live on the site. You can identify the exact version deployed by visiting the build and deployment logs available below in the status section.
+We employ various levels of integration and acceptance testing to check on the quality of the code. All our tests are done through software like Travis CI, Circle CI and Azure Pipelines.
 
-## Identifying the current version of platform
+We have unit tests for testing our challenge solutions, Server APIs and Client User interfaces. These help us test the integration between different components.
 
-**The current version of the platform is always available at [`freeCodeCamp.org`](https://www.freecodecamp.org).**
+We are also in the process of writing end user tests which will help in replicating real world scenarios like updating an email or making a call to the API or third-party services.
 
-The dev-team merges changes from the `production-staging` branch to `production-current` when they release changes. The top commit should be what you see live on the site. You can identify the exact version deployed by visiting the build and deployment logs available below in the status section.
+These tests, help in preventing issues from repeating themselves and ensure that while fixing an issue or adding a feature, we do not introduce a regression of some form.
```

Comment:
```suggestion
These tests help in preventing issues from repeating themselves and ensure that while fixing an issue or adding a feature, we do not introduce a regression of some form.
```

Reply:
Sure, I will update this. I am still adding/moving a bunch of things. The idea is soon anyone on the mods team will be able to push .dev :)

Reply:
Sorry, I could not add changes that would have enabled the mods team to push builds yet. I will get it done maybe by next week or so.